tests/app/main/views/test_index.py

```diff
@@ -25,6 +27,51 @@ def test_landing_page_404s_if_no_key_in_query_string(client):
     assert response.status_code == 404
 
 
+def test_landing_redirects_to_download_file(client):
+    service_id = uuid4()
+    document_id = uuid4()
+    key = '1234'
+    filename = "file.pdf"
+
+    response = client.get(
+        url_for(
+            'main.landing',
+            service_id=service_id,
+            document_id=document_id,
+            key=key,
+            filename=filename,
+        )
+    )
+    assert response.status_code == 302
+    assert response.headers['X-Robots-Tag'] == 'noindex, nofollow'
+    expected_url = (
+        f"http://test-doc-download-api/services/{service_id}/documents/{document_id}?key={key}&filename={filename}"
+    )
+    assert response.location == expected_url
+
+
+def test_landing_redirects_to_download_file_no_filename(client):
```

Comment:
Well that answer my last question about having no filename. I should look at tests first.